mm2src/trezor/src/transport/udp.rs

```diff
@@ -0,0 +1,160 @@
+/// This source file was borrowed from Trezor repo and modified to integrate into this project
```

Comment:
I tried to change commits to add the origin ported file to the history

Reply:
Thanks, before merging you should squash other commits except the ported code commit, then I should merge to dev without squashing so we would have 2 commits only.

Reply:
Could you please clarify, 2 or 3 commits?
in the guidelines above it is suggested to have 3 commits: initial ported code, changes to the ported code and the rest of the work

Reply:
Yeah, 3 commits is the best way :)
I thought 2 is not a big difference since the other commit will have the changes too, but 3 is better to have a seperate commit related to changes to ported the code only.